README.md

```diff
@@ -128,24 +151,26 @@ The Primary's update_cycle() call:
 - generates a Vehicle Version Manifest with some vehicle metadata and all ECU Version Manifests received from Secondaries, describing currently installed images, most recent times available to each ECU, and reports of any attacks observed by Secondaries (can also be called directly: `dp.generate_signed_vehicle_manifest()`)
 - sends that Vehicle Version Manifest to the Director (can also be called directly: `dp.submit_vehicle_manifest_to_director()`)
 
-If you wish to run the demo with multiple vehicles (one Primary each), you can open a new
+If you wish to run the demo with multiple vehicles (one Primary each), you can open a Python shell in a new terminal
 window for each vehicle's Primary and provide a unique VIN and ECU for each of them. Find the port that is chosen in the Primary's initialization and make note of it so that it can be provided to any Secondaries you set up in a moment (e.g. "Primary will now listen on port 30702")
```

Comment:
Find the port that is chosen in the Primary's initialization message (e.g., "Primary will not listen on port 30702") and make note of it so that it can be provided to any Secondaries you set up in a moment.

It might be unclear which function call initializes the primary.  Is it dp.clean_slate()?  It might not be obvious to new users from this name alone.